src/bazel-tsconfig-build.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "declaration": true,
+    "stripInternal": false,
+    "experimentalDecorators": true,
+    "noUnusedParameters": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "noImplicitAny": true,
+    "importHelpers": true,
```

Comment:
Why is `importHelpers` true?

Reply:
This has been taken from the normal `tsconfig-build.json` files. Since we want to use `tslib` for the `ng_module` output, this needs to be specified for "non-test" builds.

Even though, we might just use Bazel for serving / testing at the moment, the goal is that we publish through Bazel as well. So that's why I still kept it how it was before.